src/main/java/org/folio/rest/impl/OaiPmhViewInstancesAPI.java

```diff
@@ -80,6 +94,28 @@ public void getOaiPmhViewInstances(String startDate, String endDate, boolean del
     }
   }
 
+  @Override
+  public void postOaiPmhViewEnrichedInstances(OaipmhInstanceIds entity, RoutingContext routingContext,
+      Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+
+    UUID[] ids = entity.getInstanceids().stream().map(UUID::fromString).toArray(UUID[]::new);
+
+    getCommonInstanceProcessing(SQL_INSTANCES,
+      () -> createPostgresParams(ids, entity.getSkipSuppressedFromDiscoveryRecords()),
+      routingContext, okapiHeaders, asyncResultHandler, vertxContext,
+      "Select from oai pmh instances view completed successfully");
+  }
+
+  @Override
```

Comment:
Can you add `@Validate`? The .raml file says `is: [validate]`.

Reply:
Done